lametro/events.py

```diff
@@ -185,7 +185,7 @@ def _merge_events(self, events):
         return english_events
 
     def scrape(self, window=None) :
-        if window:
+        if window and float(window):
```

Comment:
This was a little confusing to me. What do you think about this edit to the second check, to make it more obvious why we're doing it? 

```suggestion
        if window and float(window) > 0:
```

Reply:
i think it's might be clearer to explicitly check that it's not equal to 0

Reply:
did a not equals check